include/battle_util.h

```diff
@@ -168,7 +168,7 @@ bool32 IsEntrainmentBannedAbilityAttacker(u16 ability);
 bool32 IsEntrainmentTargetOrSimpleBeamBannedAbility(u16 ability);
 
 bool32 CanSleep(u8 battlerId);
-bool32 CanBePoisoned(u8 battlerId);
+bool32 CanBePoisoned(u8 gBattlerAttacker, u8 battlerTarget);
```

Comment:
should be `bool32 CanBePoisoned(u8 battlerAttacker, u8 battlerTarget);`

Reply:
Fixed!